src/ansys/additive/core/parametric_study/parametric_study.py

```diff
@@ -1623,10 +1628,34 @@ def update_format(study: ParametricStudy) -> ParametricStudy:
             new_study._material_name = materials[0]
             version = 3
 
+            # add p/v column to the dataframe
+            df = ParametricStudy.add_pv_ratio(df)
+
         # Update the dataframe in the new study
         new_study._data_frame = df
         return new_study
 
+    @staticmethod
+    def add_pv_ratio(df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
Make this "protected" (single leading underscore) or "private" (double leading underscore). You have it as public but I don't think we want the end user calling this.